src/test/resources/com/github/checkstyle/patchfilter/BoundaryTestPatchOne.txt

```diff
@@ -0,0 +1,4 @@
+--- Origin1.java	2020-06-14 18:41:56.000000000 +0800
++++ Update1.java	2020-06-14 18:41:56.000000000 +0800
```

Comment:
Why filenames are different ?

Reply:
sorry, this is for another test, should not in this PR, I will remove it, and the reason that filenames are diff is I use `diff` command to create this patch and create two java files.